bolt/lib/nonlinear_solver/nonlinear_solver.py

```diff
@@ -132,15 +179,27 @@ def __init__(self, physical_system):
         # Assigning the function objects to methods of the solver:
         self._A_p = physical_system.A_p
 
-        # Source/Sink term(Restrict to relaxation type collision operators):
+        # Source/Sink term(restricted to relaxation type collision operators):
         self._source_or_sink = physical_system.source_or_sink
 
     def _convert_to_qExpanded(self, array):
```

Comment:
Follow uniform conventions for function naming. This should be `_convert_to_q_expanded`